wandb/sdk/internal/tpu.py

```diff
@@ -83,7 +83,12 @@ def start(self):
             self._thread.start()
 
 
-def is_tpu_available():
+def is_tpu_available() -> bool:
+    tpu_name = os.environ.get("TPU_NAME", False)
+
+    if tpu_name is False:
+        return False
+
     try:
         from tensorflow.python.distribute.cluster_resolver import tpu_cluster_resolver  # type: ignore # noqa
         from tensorflow.python.profiler import profiler_client  # type: ignore # noqa
```

Comment:
@vanpelt Chris, do you remember what these imports are for? Played safe and simply flipped the order of the checks, but still wondering if removing these for good is a good idea.